src/content/docs/infrastructure/host-integrations/host-integrations-list/port-monitoring-integration.mdx

```diff
@@ -13,8 +13,90 @@ freshnessValidatedDate: never
 
 Our port monitoring integration monitors status for networking ports, such as TCP, UDP, etc., and reports data to New Relic.
 
-To get started, see the [Port monitoring on-host integration README](https://github.com/newrelic/nri-port-monitor) on GitHub.
-
 <Callout variant="tip">
   This integration is released as [open source on Github](https://github.com/newrelic/nri-port-monitor). A change log is also available there for the latest updates. Support for this integration is available with an [Expert Services subscription](https://www.newrelic.com/expertservices).
 </Callout>
+
+## Install the integration 
+
+Follow the steps below to install the integration:
+
+<Steps>
+    <Step>
+## Install the infrastructure agent [#infra-install]
+
+To use the Port Monitoring integration, you need to first [install the infrastructure agent](/docs/infrastructure/install-infrastructure-agent/get-started/install-infrastructure-agent-new-relic/) on the same host. All our on-host integrations require the infrastructure agent, which helps expose and report metrics to New Relic.  
+
+    </Step>
+    <Step>
+## Download and prepare the integration [#download]
+
+1. Download and unpack the ZIP file from [the GitHub repo](https://github.com/newrelic/nri-port-monitor/releases):
+
+    ```shell
+    wget https://github.com/newrelic/nri-port-monitor/releases/download/1.3/nri-port-monitor.tar.gz
+    ```
+
+    ```shell
+    tar -zxvf nri-port-monitor.tar.gz 
+    ```
+
+2. Copy the `bin` directory with `nri-port-monitor` executable to `/var/db/newrelic-infra/newrelic-integrations`. Do the same for the `port-monitor-definition.yml` config file:
+
+    ```shell
+    cp nri-port-monitor/bin/port-monitor /var/db/newrelic-infra/newrelic-integrations/bin/
+    cp nri-port-monitor/port-monitor-definition.yml /var/db/newrelic-infra/newrelic-integrations/
+    ```
+
+3. Set execution permissions for the binary file `nr-port-monitor`:  
+
+    ```shell
+    chmod +x /var/db/newrelic-infra/newrelic-integrations/bin/port-monitor
+    ```
+
+4. Place the integration configuration file `port-monitor-config.yml.sample` in `/etc/newrelic-infra/integrations.d`.
+    </Step>
+    <Step>
+## Configure the integration [#configure-step]
+
+1. Rename the configuration file: 
+    ```shell
```

Comment:
what if they don't have sudo rights on their server?

Reply:
Also, this step is super weird. Aren't they doing a very similar thing in step 4 from the previous section? How are these different?

There's some useful context missing from these steps, I think